app/assets/javascripts/mirador_widget_admin.js

```diff
@@ -122,9 +152,60 @@
       return block.find('[data-behavior="mirador-item"]').length;
     }
 
+    function validManifest(manifest) {
+      if (typeof(manifest["@context"]) == "undefined") {
+        return false
+      } else if (manifest["@context"].includes('iiif.io/api/presentation/2/context.json') != true) {
+        return false
+      } else if (typeof(manifest["sequences"]) == "undefined") {
```

Comment:
This feels a bit odd to me.  I wonder if we couldn't do something like

```javascript
(manifest["@context"] || '').includes(...)
```

and that could allow us to drop a few conditionals here.  Perhaps somebody w/ more JS experience than me could weigh in about what a good pattern for doing this sort of thing would be.